.github/workflows/integration.yaml

```diff
@@ -74,7 +76,7 @@ jobs:
       max-parallel: 15
       fail-fast: false
       matrix:
-        redis-version: ['8.2.1-pre', '${{ needs.redis_version.outputs.CURRENT }}', '8.0.2' ,'7.4.4', '7.2.9']
+        redis-version: ['8.4-M01-pre', '${{ needs.redis_version.outputs.CURRENT }}', '8.0.2' ,'7.4.4', '7.2.9']
```

Comment:
I think we shouldn't change 8.2 to 8.4 and instead support both. 8.4 is another version to support so instead we have to remove 7.4 and 7.2 as soon as they EOL